builtin/aws/alb/releaser.go

```diff
@@ -304,9 +315,145 @@ func (r *Releaser) Release(
 	return &Release{
 		Url:             "http://" + hostname,
 		LoadBalancerArn: *lb.LoadBalancerArn,
+		TargetGroupArn:  target.Arn,
+		Region:          target.Region,
 	}, nil
 }
 
+func (r *Releaser) Status(
+	ctx context.Context,
+	log hclog.Logger,
+	release *Release,
+	ui terminal.UI,
+) (*sdk.StatusReport, error) {
+
+	if release.Region == "" {
+		log.Debug("Region is not available for this release. Unable to determine status.")
```

Comment:
I have a feeling that we might hit this case in the upgrade path, where someone upgrades a server then tries to promote a deployment that didn't encode the region?

Reply:
This would be good to test, sounds like a correct assumption.